lib/Plack/Handler/Apache2.pm

```diff
@@ -144,6 +144,8 @@ sub fixup_path {
         $env->{SCRIPT_NAME} = '';
     } elsif ($path_info =~ s{^($location)/?}{/}) {
         $env->{SCRIPT_NAME} = $1 || '';
+    } elsif ( $r->filename() =~ /$location/) {
```

Comment:
i think this match has a false positive.

Reply:
could be, probably begin anchor would be enough. I'll have a look at it again tomorrow.

Reply:
so, on second glance, no, it cannot have false positive. location is the regexp <Files> has and it seems a bit redundant here, because if we reach here, /$location/ will always match and will be correct. so I don't believe there is a problem here.

Reply:
any eta for merge? tnx.